app/mailers/registration_mailer.rb

```diff
@@ -4,7 +4,7 @@ class RegistrationMailer < ApplicationMailer
   def confirmation_email
     user = params[:user]
     @user = user
-    @url = "#{root_url}/api/v1/users/confirmation?email_token=#{params[:email_token]}"
+    @url = "http://#{Rails.application.config.hosts.first}/api/v1/users/confirmation?email_token=#{params[:email_token]}"
```

Comment:
Please, don't hardcode it. You should think about how to write more flexible.

```ruby
def confirmation(email, token, path)
    @email = email
    @confirmation_url = URI.parse("#{path}?email_token=#{token}").to_s
    mail(to: email, subject: I18n.t('user_mailer.confirmation.subject'))
  end
```

Reply:
Refactored